R/outputs.R

```diff
@@ -763,25 +783,27 @@ save_output <- function(filename, dir,
     write_datapack_csv(naomi_output, "pepfar_datapack_indicators_2022.csv")
   }
 
-  dir.create("info")
-  write_navigator_checklist(naomi_output, "info/unaids_navigator_checklist.csv")
 
+  dir.create("info")
   info <- attr(naomi_output, "info")
+
   if (length(info) > 0L) {
     for (p in names(info)) {
       writeLines(trimws(info[[p]]), file.path("info", p))
     }
   }
 
+  write_navigator_checklist(naomi_output,
```

Comment:
This will fail for old model outputs I think (because data options are not available) - probably with a not very useful error. We should check if the `naomi_output` has the appropriate format to create the checklist. If not then don't save this file out, and maybe display a warning to the user?

Reply:
We need a test that checks this case too

Reply:
Thanks -- this is addressed in 10e2441.  The behaviour is to still write the file, but `NA` values are returned for checks that cannot be checked due to missing options list. 

Tests are here: https://github.com/mrc-ide/naomi/blob/navigator-checklist/tests/testthat/test-outputs.R#L320-L387